.github/workflows/gosec.yml

```diff
@@ -35,11 +35,7 @@ jobs:
           cache: false
       - name: Install Gosec
         run: go install github.com/securego/gosec/v2/cmd/gosec@latest
-      - name: Run Gosec (root)
-        working-directory: .
-        run: |
-          gosec -exclude-dir=arangodb -exclude-dir=badger -exclude-dir=couchbase -exclude-dir=dynamodb -exclude-dir=etcd -exclude-dir=memcache -exclude-dir=memory -exclude-dir=mongodb -exclude-dir=mysql -exclude-dir=postgres -exclude-dir=redis -exclude-dir=ristretto -exclude-dir=sqlite3 -exclude-dir=s3 -exclude-dir=bbolt -exclude-dir=azureblob -exclude-dir=mssql -exclude-dir=pebble ./....
```

Comment:
instead of removing this we could just run it for root or ? with "." at the end
`gosec .`
then we don't have to customize anything in the future and it still checks the code in the root

Reply:
We can run `gosec .` since it gets confused with all the go.mod files.

Reply:
but locally it worked for me
it then takes only the root folder and no subfolders

Reply:
Done